variables.tf

```diff
@@ -98,3 +98,15 @@ variable "infrastructure_execution_count" {
   type = number
   default = 1
 }
+
+variable "number_of_controller_instances" {
```

Comment:
I named these slightly different for eda and execution based on the way the example I had looked at was naming stuff. We will need to come to a consensus and be consistent.

Reply:
complete